lib/events/filesessions/fileasync.go

```diff
@@ -294,12 +304,19 @@ func (u *Uploader) periodicSpaceMonitor(ctx context.Context) {
 			if usedPercent > float64(events.DiskAlertThreshold) {
 				u.log.WarnContext(ctx, "Free disk space for audit log is running low", "percentage_used", usedPercent)
 				if u.cfg.AlertHandler != nil {
+					instanceType := fmt.Sprintf("node %q", u.cfg.Hostname)
+					if slices.Contains(u.cfg.SystemRoles, types.RoleAuth) {
```

Comment:
nit: you can use the `types.SystemRoles` type instead and call `u.cfg.SystemRoles.Include(types.RoleAuth)`